### PR TITLE
VMS: don't use /DSF, turn off CALL_DEBUG instead

### PR DESCRIPTION
**Pull request by levitte**
It turns out that /DSF didn't do any good for our purposes.  Instead,
remove the CALL_DEBUG flag from any image we link.  This ensures that
we can have debugging information in the image files, but don't
automatically end up in a debugging session upon image activation.

Unfortunately, this means the CALL_DEBUG must be turned on when there
is a need to run with the debugger activated, and to turn it off when
done.  This has been documented in NOTES.VMS.

### CODE REVIEW

File: util/openssl-format-source [hunk 1]
```diff
@@ -75,7 +75,7 @@ do
   fi
 
   if [ -d "$i" ]; then
-    LIST=`find "$i" -name '*.[ch]' -print`
+    LIST=`find "$i" -name '*.[ch]' -o -name '*.cc' -print`
```
Comment (richsalz) [inline util/openssl-format-source]: seems unrelated to this PR

**Review by richsalz — CHANGES_REQUESTED** [review]

File: util/openssl-format-source (hunk 1 above)
Comment (levitte) [inline util/openssl-format-source]: Oops, yup.

**Comment by levitte** [comment]
`uitl/openssl-format-source` change removed

**Review by richsalz — APPROVED** [review]

**Comment by levitte** [comment]
Merged!  1ae4c07e50a4988c7e2263ad669dc75fe2d494cb

Thanks
